lib/index.js

```diff
@@ -27,6 +27,8 @@ import {
   createAccount
 } from './actions';
 
+import { safeSet } from './utilities';
```

Comment:
Do you see this being useful in other situations? Does it make sense to add to bpanel-utils? or is it too specific? Almost feels like you could publish it as it's own stand alone npm module! :D

Reply:
Thanks! It could be useful in `bpanel-utils` if we test it with more edge cases